build/windows/download_nrjmx.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+
+# Gets last version of nrjmx MSI installer
+
+echo "Downlading last version of nrjmx MSI installer"
+if [[ -z $NRJMX_URL ]]; then
+    NRJMX_VERSION=$(curl --silent "https://api.github.com/repos/newrelic/nrjmx/releases/latest" | grep '"tag_name":' |  grep -oE '[0-9.?]+')
+    echo "Using latest nrjmx version $NRJMX_VERSION."
+    NRJMX_URL=https://github.com/newrelic/nrjmx/releases/download/v$NRJMX_VERSION/nrjmx-amd64.$NRJMX_VERSION.msi
+fi
+
+curl "$NRJMX_URL" -o "build/package/windows/bundle/nrjmx-amd64.msi"
```

Comment:
I'd add --show-error and --fail flags just in case the url doesn't point to a valid package

Reply:
> looks good! Do you know how would be the behavior of installing another integation with the same dependency for instance nri-cassandra?

This is a good point. I haven't been able to test this since we only have one bundle for now. I'll open another PR on a different integration and try it.

Reply:
Done!